django/db/backends/base/schema.py

```diff
@@ -732,6 +738,15 @@ def _alter_field(self, model, old_field, new_field, old_type, new_type,
             fragment = self._alter_column_null_sql(model, old_field, new_field)
             if fragment:
                 null_actions.append(fragment)
+
+        if old_field.db_comment != new_field.db_comment:
+            fragment, other_actions = self._alter_column_comment_sql(model, new_field, new_type, new_field.db_comment)
+            # postgres, oracle can't execute same time 'ALTER COLUMN ...' and 'COMMENT ON ...'
```

Comment:
I'm not exactly sure why this comment is here, I don't think it adds to readability. If we keep it, we can improve the English a bit:

```suggestion
            # Postgres and Oracle can't execute 'ALTER COLUMN ...' and 'COMMENT ON ...' at the same time.
```